src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts

```diff
@@ -66,14 +69,30 @@ export class CommandsQuickAccessProvider extends AbstractEditorCommandsQuickAcce
 		@IConfigurationService private readonly configurationService: IConfigurationService,
 		@IEditorGroupsService private readonly editorGroupService: IEditorGroupsService,
 		@IPreferencesService private readonly preferencesService: IPreferencesService,
+		@IProductService productService: IProductService
 	) {
+		const suggestCommands = configurationService.getValue<boolean>('workbench.commandPalette.experimental.suggestCommands');
```

Comment:
@TylerLeonhardt I feel there is now a bit of an ugly repetition of the same code to handle the suggested commands. What I typically do in these cases is to extract the config handler into a method and then call it twice: once initially and once from the event. The event itself then becomes optional, see for example:

https://github.com/microsoft/vscode/blob/d8568bb19e5acb54253afb2cb48123785db2f8d9/src/vs/workbench/browser/workbench.ts#L264